backend/lib/components/Auth0.js

```diff
@@ -85,9 +85,49 @@ const getUser = async (token) => {
   }
 };
 
+const getUserByEmail = async (token, email) => {
```

Comment:
I'm no longer using this but is it OK to leave it here for future use? It could be useful.

Reply:
I prefer to delete unused code (YAGNI) and we still have this commit for reference if we need it later
But most of the time I think we will probably use the user id or token
Also, ESLint might complain?
Besides, this `components/` folder originally started from the first version using Express and Passport middleware... and I don't really like the structure (like with the `wrapError` etc. which was a bit of a workaround that I added later to patch it... ) so I think it's better to keep this file minimal and maybe refactor the backend later.

Reply:
Ooh true, ESlint does complaint if declared but unused. There is a ticket to clean up ESLint messages...there's too much noise atm. I'll take care of that.